src/main/java/gtclassic/recipe/GTRecipeUUAmplifier.java

```diff
@@ -49,14 +48,12 @@ public static void init() {
 	}
 
 	public static void addAmplifierToJei(IRecipeInput[] inputs, IRecipeModifier[] modifiers, ItemStack... outputs) {
-		List<IRecipeInput> inlist = new ArrayList<>();
-		List<ItemStack> outlist = new ArrayList<>();
-		Collections.addAll(inlist, inputs);
+		List<IRecipeInput> inlist = ObjectArrayList.wrap(inputs);
+		List<ItemStack> outlist = ObjectArrayList.wrap(outputs);
 		NBTTagCompound mods = new NBTTagCompound();
 		for (IRecipeModifier modifier : modifiers) {
 			modifier.apply(mods);
 		}
-		Collections.addAll(outlist, outputs);
 		addAmplifierToJei(inlist, new MachineOutput(mods, outlist));
```

Comment:
make it direct xD

Reply:
fixed in 4b65ed77f80277727fce24f4501157398784c45a